ideascube/mediacenter/urls.py

```diff
@@ -4,7 +4,6 @@
 
 urlpatterns = [
     url(r'^$', views.index, name='index'),
-    url(r'^tag/(?P<tag>[\w\-_]+)/$', views.by_tag, name='by_tag'),
```

Comment:
Open question: do we need to take care of retrocompatibility of this URL?
I'd say no, but let's keep that in mind.